src/components/common/Tour.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import IconButton from '@material-ui/core/IconButton';
+import InfoIcon from '@material-ui/icons/Info';
+import JoyRide, { ACTIONS, EVENTS, STATUS } from 'react-joyride';
+import { TOUR_STEPS_EN, TOUR_STEPS_FR } from '../../config/tour-steps';
+import { postAppInstanceResource } from '../../actions';
+import { TOUR_TAKEN } from '../../config/appInstanceResourceTypes';
+import { FRENCH_LANGUAGE_STRING } from '../../config/constants';
+
+const Tour = () => {
+  const dispatch = useDispatch();
+  const { standalone, lang } = useSelector(({ context }) => context);
+  const { content: appInstanceResourcesContent } = useSelector(
+    ({ appInstanceResources }) => appInstanceResources,
+  );
+  const INITIAL_TOUR_STATE = {
+    run: false,
+    continuous: true,
+    loading: false,
+    stepIndex: 0,
+    steps: lang === FRENCH_LANGUAGE_STRING ? TOUR_STEPS_FR : TOUR_STEPS_EN,
```

Comment:
See below about making dynamic for whatever language. In this case, you could probably use the `i18n` package where you define the steps.

Reply:
@hasanagh can provide an example, as I believe we did this for desktop.

Reply:
will be addressed in #59